src/Modules/Modal.react.js

```diff
@@ -183,11 +185,11 @@ class Modal extends Component {
                                 style={containerInnerScrollStyles}
                             >
                                 {header ? React.Children.map(this.props.children, c => React.cloneElement(c, {
-                                    closeButton: this.props.closeButton,
-                                    inverse: this.props.inverse,
+                                    closeButton: closeButton,
+                                    inverse: inverse,
                                     onClose: this._onClose,
-                                    title: this.props.title,
-                                    titleTruncate: this.props.titleTruncate,
+                                    title: title,
+                                    titleTruncate: titleTruncate,
```

Comment:
Similar here ... we can short-hand this ever further if we really want, but that's definitely in super nit-pick land.